index.js

```diff
@@ -1,12 +1,18 @@
 const express = require("express");
+const swaggerUi = require("swagger-ui-express");
+const swaggerDocument = require("./src/swagger");
 
 // express app
 const app = express();
 const router = require("./src/routes");
 
+
 require("./start/config")(app, express);
 require("./start/db")(app);
 require("./start/logging")();
 
+// swagger
```

Comment:
According to the project structure, the middleware is added to the start folder

Reply:
ok

Reply:
fixed changes, please merge

Reply:
It is only prepared for the article model, right?

Reply:
yes, add other model in future